lib/linter-crystal.js

```diff
@@ -50,16 +50,18 @@ export default {
         args.push(activeEditor.getPath())
         resolve(helpers.exec(command, args).then(output => {
           const result = []
-          for (issue of JSON.parse(output)) {
-            result.push({
-              type: "error",
-              text: issue.message,
-              filePath: issue.file,
-              range: [
-                [issue.line - 1, issue.column -1 ],
-                [issue.line - 1, issue.column + issue.size - 1]
-              ]
-            })
+          if (output) {
```

Comment:
Instead of indenting all of the code, You could just do `if (!output) return result;`